conda-recipe/rsmtool/meta.yaml

```diff
@@ -22,19 +21,35 @@ build:
     - render_notebook = rsmtool.reporter:main
     - convert_feature_json = rsmtool.convert_feature_json:main
 
-{% block requirements -%}
 requirements:
   build:
-    - python >=3.6
```

Comment:
Was this gotten rid of only due to the numpy issue?

Reply:
Not really. It was because we don't need to specify a version anymore since we are building specific variants using `conda_build_config.yaml`.